src/server/goog-device/mw/WebsocketProxyOverAdb.ts

```diff
@@ -269,15 +287,17 @@ export class WebsocketProxyOverAdb extends WebsocketProxy {
                                         this.logger.info(`Success to swipe: ${xx} ${y1} ${xx} ${y2} 500`);
                                     });
                             })
-                            .catch((error) => {
-                                this.logger.error(error);
+                            .catch((e) => {
+                                e.message = 'Failed to swipe';
+                                this.logger.error(e);
                             });
                         return;
                     }
                 }
             }
-        } catch (error) {
-            this.logger.error(error);
+        } catch (e) {
+            this.logger.error(e);
+            Utils.captureMessage(e.message, 'Android', this.udid);
```

Comment:
@metocks // iOS쪽과 동일하게 수정해 보았습니다.
```suggestion
            const self = this;
            Sentry.captureException(e, scope => {
                 scope.setTags('ramiel_device_type', 'Android');
                 scope.setTags('ramiel_device_id', self.udid);
                 scope.setTags('ramiel_message', e.ramiel_message);
                 return scope;
            });
```